pkg/controller/util.go

```diff
@@ -91,6 +91,23 @@ func getSecretName(client kubernetes.Interface, pvc *v1.PersistentVolumeClaim) (
 	return name, nil
 }
 
+// returns the import content.
```

Comment:
// returns the import content.  --- well, not really; it's actually doing a `getOrSetContentType`